packages/xo-server/src/api/xosan.js

```diff
@@ -821,12 +821,14 @@ export const createSR = defer(async function(
 createSR.description = 'create gluster VM'
 createSR.permission = 'admin'
 createSR.params = {
+  brickSize: { type: 'number', optional: true },
   srs: {
     type: 'array',
     items: {
       type: 'string',
     },
   },
+  template: { type: 'object' },
```

Comment:
How do you know this is an object?

Reply:
By its usage
https://github.com/vatesfr/xen-orchestra/blob/f0e4004fbff22ec00901d59ce1a0a3f3a2609682/packages/xo-server/src/api/xosan.js#L1169-L1170